### PR TITLE
Update MIP46.md

### PR DESCRIPTION
**Pull request by tylerrihn**
@blimpa howdy :) I made some edits for grammar/formatting for this MIP46; nothing big! Give a look and merge to master if you think everything is looking alright.

MIP46 Tracked Changes: Parameter Proposal Groups
-	Updated Dating Structure to reflect ISO standard YYYY, MM,DD

Sentence Summary
-	2nd line, capitalize “Contributors” and Domain Teams
-	Change Domain Teams to Core Units??
-	Capitalize Parameter Change Proposals
Paragraph Summary
-	Capitalize Contributors and Domain Teams
-	Capitalize Parameter Change Proposals
-	Capitalize Prameter Adjustment Proposals
Mip46c3
-	Capitalize Parameter Proposals
Motivation
-	Remove comma after “complexity”, first sentence.

Specification/Proposal Details
-	Capitalize “Parameter Proposal Group”, first paragraph

Mip46c2 Parameter Proposal Requirements:
-	Capitalize Parameter Proposal, first sentence
Mip46c3: Parameter Proposal Process:
-	Capitalize Weekly Governance Cycle
-	Grammar Change 3rd Sentence, replace “, and” with a period.

### CODE REVIEW

File: MIP46/MIP46.md [hunk 1]
```diff
@@ -90,10 +90,10 @@ Each parameter proposal must fulfill a set of requirements in order to be recogn
...
+Parameter Proposals are first presented on the Maker Governance forum and are then entered into the Weekly Governance Cycle by the Governance Facilitators. Where possible, Parameter Proposals should be made public on the first Monday of the month for inclusion into the weekly cycle in the second week of the month. 
 
 Sufficient time must be given for discussion, and inclusion in the weekly cycle may be delayed by the Governance Facilitators if they feel more discussion is required.
 
-Proposed Parameter Changes will be bundled into a single on-chain poll for inclusion in the weekly cycle, and if the outcome is favorable, will proceed to that week's executive vote absent unforeseen circumstances.
+Proposed Parameter Changes will be bundled into a single on-chain poll for inclusion in the weekly cycle. If the outcome is favorable, it will proceed to that week's executive vote absent unforeseen circumstances.
```
Comment (blimpa) [inline MIP46/MIP46.md]: Capitalize "weekly cycle" here

**Review by blimpa — CHANGES_REQUESTED** [review]

File: MIP46/MIP46.md (hunk 1 above)
Comment (tylerrihn) [reply MIP46/MIP46.md]: yessir

**Comment by blimpa** [comment]
LGTM :)

**Review by blimpa — APPROVED** [review]
Thank you!

**Pull request merged**